server.rb

```diff
@@ -9,11 +9,11 @@ class SealApp < Sinatra::Base
   get '/' do
     "Hello Seal"
   end
-  # POST /teams/core-formats
 
-
-  post '/teams/?:team_name?' do
-    Seal.new(params[:team_name]).bark
+  get '/:team_name/:secret' do
```

Comment:
What would you think of making `/:team_name/bark`? You might need the `/:team_name` route later to get the configs for a specific team. It also makes the hubot-side self-documenting.

Did you have a problem with doing `POST`?